runtime/hsa-runtime/inc/hsa_ext_amd.h

```diff
@@ -2828,6 +2829,102 @@ typedef enum hsa_amd_queue_priority_s {
 hsa_status_t HSA_API hsa_amd_queue_set_priority(hsa_queue_t* queue,
                                                 hsa_amd_queue_priority_t priority);
 
+/**
+ * @brief Queue creation attributes.
+ */
+typedef enum {
+  /**
+   * The queue packet buffer and the queue struct should be allocated in
+   * the agent's device memory.
+   */
+  HSA_AMD_QUEUE_FLAG_DEVICE_MEM = (1 << 0),
```

Comment:
We should explicitly mention whether its cached on uncached. Like HSA_AMD_QUEUE_HOST = 0, HSA_AMD_QUEUE_DEV_UNCACHED = 1 << 0,

Reply:
Removed the cache flag for now.